staker/staker.go

```diff
@@ -325,6 +332,13 @@ func NewStaker(
 		if err != nil {
 			return nil, err
 		}
+		isOwner, err := fastConfirmSafe.safe.IsOwner(&callOpts, wallet.AddressOrZero())
+		if err != nil {
+			return nil, err
+		}
+		if !isOwner {
+			log.Info("fast confirmer is not part of owners of safe", "fastConfirmer", fastConfirmer, "wallet", wallet.AddressOrZero())
```

Comment:
Shouldn't this also disable fast confirmations to avoid excess txs for approval?

Reply:
Done.